rules/form-components-must-be-wrapped.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+const Rule = require('ember-template-lint').Rule
+
+module.exports = class FormComponentsMustBeWrapped extends Rule {
+  INVALID_TAG_NAMES = [
+    'ui-text-field',
+    'UiTextField',
+    'ui-address-autocomplete',
+    'UiAddressAutocomplete',
+    'ui-checkbox-standalone',
+    'UiCheckboxStandalone',
+    'ui-date-time-picker',
+    'UiDateTimePicker',
+    'ui-radio-group',
+    'UiRadioGroup',
+    'ui-select',
+    'UiSelect',
+    'ui-signature-pad',
+    'UiSignaturePad',
+    'ui-text-area',
+    'UiTextArea',
+    'ui-segmented-control',
+    'UiSegmentedControl'
+
+    // ui-checkbox and ui-toggle currently don't work using form objects but should be added to this list
+    // when they are updated
```

Comment:
How will people know to add new form components to this list?

Reply:
this would probably be something we'd have to monitor for now. I could add a task to update this in the EPIC. eventually if I have time I'd like to get these rules into TAG and then we could automate this by checking which components extend off of `FormFieldComponent`

Reply:
https://financeit.atlassian.net/browse/FEN-834